ucdn.h

```diff
@@ -346,6 +350,25 @@ int ucdn_get_mirrored(uint32_t code);
 uint32_t ucdn_mirror(uint32_t code);
 
 /**
+ * Get paired bracket for a codepoint.
+ *
+ * @param code Unicode codepoint
+ * @return paired bracket codepoint or the original codepoint if no
+ * paired bracket character exists
+ */
+uint32_t ucdn_paired_bracket(uint32_t code);
+
+/**
+ * Get paired bracket type for a codepoint.
+ *
+ * @param code Unicode codepoint
+ * @return value according to UCDN_PAIRED_BRACKET_TYPE_* and as defined
+ * in UAX#9.
+ *
+ */
+int ucdn_paired_bracket_type(uint32_t code);
+
+/**
```

Comment:
Also, I'm not sure about the API here. Maybe you need both the bracket type and thei paired bracket itself? Just a thought.